packages/paste-website/stories/PageTemplates.stories.tsx

```diff
@@ -860,3 +880,505 @@ export const SettingsOverviewExample = (): JSX.Element => {
 SettingsOverviewExample.parameters = {
   padding: false,
 };
+
+export const DefaultWizardExample = (): JSX.Element => {
+  const input1 = useUID();
+  const input2 = useUID();
+  const input3 = useUID();
+  const input4 = useUID();
+  return (
+    <Box display="flex" width="100%" justifyContent="center" paddingX="space10">
+      <Box
+        paddingTop="space130"
+        paddingBottom="space160"
+        display="flex"
+        flexDirection="column"
+        rowGap="space130"
+        width="size70"
+        maxWidth="size80"
+      >
+        <Box display="flex" flexDirection="column">
+          <Box display="flex" rowGap="space30" marginBottom="space130">
+            <Breadcrumb aria-label={useUID()}>
+              <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+              <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+              <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+            </Breadcrumb>
+          </Box>
+          <ProgressSteps>
+            <ProgressStepCurrent as="div">Label</ProgressStepCurrent>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+          </ProgressSteps>
+          <Box marginTop="space70">
+            <DetailText>Wizard title</DetailText>
+            <Heading as="h4" variant="heading10">
+              Verb heading
+            </Heading>
+            <Paragraph marginBottom="space0">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi non ex risus. Aenean at ante id lectus
+              faucibus hendrerit a fringilla lorem. Pellentesque faucibus sit amet dolor vitae lacinia.
+            </Paragraph>
+          </Box>
+        </Box>
+        <Box>
+          <Form>
+            <FormControl>
+              <Label htmlFor={input1}>Label</Label>
+              <Input type="text" id={input1} name="input1" />
+            </FormControl>
+            <FormControl>
+              <Label htmlFor={input2}>Label</Label>
+              <Input type="text" id={input2} name="input2" />
+            </FormControl>
+            <FormControl>
+              <Label htmlFor={input3}>Label</Label>
+              <Input type="text" id={input3} name="input3" />
+            </FormControl>
+            <FormControl>
+              <Label htmlFor={input4}>Label</Label>
+              <Input type="text" id={input4} name="input4" />
+            </FormControl>
+          </Form>
+        </Box>
+        <Box display="flex" justifyContent="end" columnGap="space40">
```

Comment:
suggestion/question: Use ButtonGroup here to get the default margin? Are there any downsides to using Button Group in general?

Reply:
no downside just forgot :)